packages/devtools_app/lib/src/inspector/inspector_screen.dart

```diff
@@ -104,6 +145,26 @@ class InspectorScreenBodyState extends State<InspectorScreenBody>
         isSummaryTree: true,
       ),
     );
+
+    final treeColumn = ValueListenableBuilder<List<InspectableWidgetError>>(
```

Comment:
how does this look for the narrow width version of the inspector? We may need some UX input on how to best display this information - especially since this change would likely be part of the demo I am recording soon. @InMatrix @cobblest

Reply:
I just took a look at the screenshot in #2503. I'm curious about the intended workflow for the user. How can someone go from seeing the badge of error count on the inspector tab to locating the error-causing widget?

Reply:
I had the same question. From the screenshot it might be difficult for users to find the error-causing widget. It would be great if we could highlight the problematic widget in the widget tree like the prototype here: https://docs.google.com/presentation/d/1Ocgvw9RP_mL657ACSmGVs0RRiPGuacGbAK97KAtE0eM/edit#slide=id.g7edbfe90b0_2_47

Reply:
@kenzieschmoll 

> how does this look for the narrow width version of the inspector?

Looks like this:

![Screenshot 2021-01-27 at 17 37 28](https://user-images.githubusercontent.com/1078012/106030717-57eef000-60c6-11eb-89f9-0fb0dc63a7eb.png)

I'm a little unsure if this is how we wanted to render errors (I was after some feedback) - I assumed we need a list somewhere, because the errored widgets may be offscreen (or perhaps collapsed?). I think the widgets should still be badged something like the mockup above.

@InMatrix @cobblest

> I just took a look at the screenshot in #2503. I'm curious about the intended workflow for the user. How can someone go from seeing the badge of error count on the inspector tab to locating the error-causing widget?

The number in the badge on the Inspector tab corresponds to the number of errors rendered in the bottom left of that screenshot. You can click on them to jump to the relevant widget in the tree. The widgets should also be marked visibily in the tree, though that's not done here yet.

Reply:
Thanks for explaining the interaction. It sounds good to me. Looking forward to trying it out live.

Reply:
i've created some mocks to put the prototype into better context @cobblest posted earlier: 

Note:  Using the error color in the screenshot. **Final error color should match established color scheme.** This is FPO.

- Design 1A has a marker on the Row and the Overflow error.  
_Intention:_  if there's an issue at the Row level, the Row is badged when collapsed.  When open, the badged marker identifies the overflow error and the Row level badge is not visible. This marker appears when the Row is collapsed and the error has not been resolved.
- Design 1 + 6 have the difference of the badge marker by the overflow error.  
_Intention:_  Using the badge markers to navigate to the overflow error quickly without needing to scroll within the tree.  This helps visualize the user experience without a marker with an error.  Marker will help surface the error if the overflow widget is nested within a deep tree.


![devtools-le-error-0a1](https://user-images.githubusercontent.com/8105802/106048200-6bd02d00-6099-11eb-8abf-2c59238ea9b6.png)

![devtools-le-error-01](https://user-images.githubusercontent.com/8105802/106046008-8e147b80-6096-11eb-9624-86ea4992c10a.png)
![devtools-le-error-02](https://user-images.githubusercontent.com/8105802/106046012-8f45a880-6096-11eb-946e-7076f6675da8.png)
![devtools-le-error-03](https://user-images.githubusercontent.com/8105802/106046014-9076d580-6096-11eb-8143-5d9272f27f33.png)
![devtools-le-error-04](https://user-images.githubusercontent.com/8105802/106046015-910f6c00-6096-11eb-8eda-9c9bf92e7035.png)
![devtools-le-error-05](https://user-images.githubusercontent.com/8105802/106046016-91a80280-6096-11eb-83f9-5a6ce2011c38.png)
![devtools-le-error-06](https://user-images.githubusercontent.com/8105802/106046020-92409900-6096-11eb-8443-f4d7a6753459.png)